prime-router/src/main/resources/db/migration/V20__add_sftp_check_result_table.sql

```diff
@@ -0,0 +1,35 @@
+/*
+ * The Flyway tool applies this migration to create the database.
+ *
+ * Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+ * use VARCHAR(63) for names in organization and schema
+ *
+ * Copy a version of this comment into the next migration
+ *
+ */
+
+CREATE TABLE receiver_connection_check_results
+(
+  connection_check_result_id BIGSERIAL PRIMARY KEY,
+  organization_id INTEGER NOT NULL REFERENCES setting(setting_id) ON DELETE CASCADE,
+  receiver_id INTEGER NOT NULL REFERENCES  setting(setting_id) ON DELETE CASCADE,
+  connection_check_result TEXT NOT NULL,
+  connection_check_successful BOOLEAN NOT NULL DEFAULT FALSE,
+  connection_check_started_at TIMESTAMP WITH TIME ZONE NOT NULL,
```

Comment:
I wonder if timestampz is better.  Not sure we care about the timezone since it will be Zulu in Azure anyways